addons/hr_expense/models/hr_expense_sheet.py

```diff
@@ -657,6 +657,11 @@ def _check_can_create_move(self):
         if any(not sheet.journal_id for sheet in self):
             raise UserError(_("Specify expense journal to generate accounting entries."))
 
+        if False in self.mapped('payment_mode'):
```

Comment:
```suggestion
        if not all(self.mapped('payment_mode')):
```
nitpick: also, not sure about this, but perhaps it should be `the employee`, not `an employee`?